self_enrollment_tool/views.py

```diff
@@ -123,10 +211,8 @@ def enable (request, course_instance_id):
     logger.debug(request)
     role_id = request.POST.get('role_id')
 
-    # todo: retrive role name. Note: Thsi is implemented in teh other story for thsi tool but will remove this comment
-    #  after verifying on merging upstream
-    role_name = request.POST.get('role_id')
-    logger.debug(f'Selected Role_id {role_id} for Self Enrollment in  course {course_instance_id}')
+    role_name = request.POST.get('role_name')
```

Comment:
Here we probably want to be using the role ID rather than the name.

Reply:
We are also getting role ID in a request above. Let me know if we want to change that